src/Servers/Kestrel/Core/src/Internal/Infrastructure/TransportManager.cs

```diff
@@ -68,7 +68,7 @@ public async Task<EndPoint> BindAsync(EndPoint endPoint, MultiplexedConnectionDe
             _transports.Add(new ActiveTransport(connectionListener, acceptLoopTask, transportConnectionManager, endpointConfig));
         }
 
-        public Task StopEndpointsAsync(List<EndpointConfig> endpointsToStop, CancellationToken cancellationToken)
+        public Task StopEndpointsAsync(List<EndpointConfig?> endpointsToStop, CancellationToken cancellationToken)
```

Comment:
Why did this change? I put `#nullable enable` at the top of this already. I don't think endpointsToStop can contain null.